src/main/java/org/jabref/gui/entryeditor/FieldsEditorTab.java

```diff
@@ -179,7 +180,8 @@ private Region setupPanel(JabRefFrame frame, BasePanel bPanel, boolean compresse
             gridPane.setStyle(
                     "text-area-background: " + convertToHex(GUIGlobals.validFieldBackgroundColor) + ";"
                             + "text-area-foreground: " + convertToHex(GUIGlobals.editorTextColor) + ";"
-                            + "text-area-highlight: " + convertToHex(GUIGlobals.activeBackgroundColor) + ";");
+                            + "text-area-highlight: " + convertToHex(GUIGlobals.activeBackgroundColor) + ";"
+            + "text-area-font-size: " + Globals.prefs.getInt(JabRefPreferences.MENU_FONT_SIZE) + "pt;");
```

Comment:
adding `+ "-fx-font-size: 20pt;"` here without any other code changes seems to do the trick.